apps/archive/archive.py

```diff
@@ -443,8 +444,10 @@ def deschedule_item(self, updates, original):
         :param original: original is document.
         """
         updates[ITEM_STATE] = CONTENT_STATE.PROGRESS
-        updates[PUBLISH_SCHEDULE] = None
-        updates[SCHEDULE_SETTINGS] = {}
+        # we need to convert schedule time to UTC
+        tz = pytz.timezone(original[SCHEDULE_SETTINGS]['time_zone'] or 'UTC')
+        localized = tz.localize(original['publish_schedule'].replace(tzinfo=None))
+        updates[PUBLISH_SCHEDULE] = localized.astimezone(pytz.utc)
```

Comment:
My opinion is that the changes on lines 448, 449, 450 may be unnecessary. We just need to keep whatever is in PUBLISH_SCHEDULE and SCHEDULE_SETTINGS fields, don't need to re-assign them.